e2e/tests/ui-driven/.eslintrc

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": ["../../.eslintrc", "plugin:playwright/playwright-test"]
+  "extends": ["../../.eslintrc", "plugin:playwright/playwright-test"],
+  "rules": {
+    "playwright/expect-expect": [
+      "error",
+      {
+        "additionalAssertFunctionNames": ["expectSections"]
+      }
+    ],
+    "playwright/no-networkidle": "warn"
```

Comment:
This one is a bit of a known outstanding issue - ideally we wouldn't rely on the `networkidle` status but I didn't want to get sidetracked into unpicking this just now (and it's unrelated to failure on other branch).

Docs: https://playwright.dev/docs/api/class-page#page-wait-for-load-state